src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/TalismanListener.java

```diff
@@ -138,7 +139,8 @@ public void onEnchant(EnchantItemEvent e) {
 			}
 			
 			String enchant = enchantments.get(random.nextInt(enchantments.size()));
-			e.getEnchantsToAdd().put(Enchantment.getByKey(NamespacedKey.minecraft(enchant.split("-")[0])), Integer.parseInt(enchant.split("-")[1]));
+			String[] enchantSplit = PatternUtils.SLASH_SEPARATOR.split(enchant);
```

Comment:
This should also rather be DASH than SLASH right?

Reply:
fixed